scripts/commands/wizard/quiz.js

```diff
@@ -0,0 +1,187 @@
+      {
+        type: 'input',
+        name: 'incorrectFeedback',
+        message: 'What failure message should we show the learner if they pick this option? Be sure to explain why this choice is wrong so they can feel more confident when choosing again.',
+        validate: validateStringPresent
+      }
+    ])
+
+    wrongAnswer = {
+      answer: responses2.incorrectAnswer,
+      correct: false,
+      feedback: responses2.incorrectFeedback
+    }
+
+    log.info(`Your new wrong answer is: "${wrongAnswer.answer}"`)
+    log.info(`The feedback provided will be: "${wrongAnswer.feedback}"`)
+
+    choices.push(wrongAnswer)
+
+    log.info(`You currently have 1 correct answer and ${choices.length - 1} wrong answers. (We recommend providing 2-3 incorrect options.)`)
+
+    if (!(await promptRepeat('wrong answer'))) {
+      askAgain = false
+    }
+  } // end while loop
+  // shuffle right and wrong answers to change position of correct one
+  for (let i = (choices.length - 1); i > 0; i--) {
+    const j = Math.floor(Math.random() * i)
+    const temp = choices[i]
+    choices[i] = choices[j]
+    choices[j] = temp
+  }
+
+  logList(`Cool! Here's what we get when we mix up the order of the answer choices:`, choices.map((choice, index) => `Option ${index + 1}: \n    ‣ Answer: ${choice.answer} \n    ‣ Feedback [${choice.correct ? 'Correct' : 'Incorrect'}]: ${choice.feedback}`))
+  await api.lessons.updateQuiz(tutorial, lesson, { question, choices })
+  afterQuizCreate(tutorial, lesson, { createLesson, createTutorial, createResource })
+} // end createQuiz
+
+// *** TRANSITIONAL DIALOGS & PROMPTS ***
+
+async function createQuizIntro ({ createLesson, createTutorial, createResource }) {
+  log.info("Let's create a multiple-choice quiz! This will only work if you've already created the tutorial and lesson files.")
+  if (await promptFilesReady()) {
+    const tutorial = await selectTutorial('quiz', { createTutorial, createResource, createLesson, createQuiz })
+    // CHOOSE A LESSON
+    const lesson = await selectLesson(tutorial)
+    if (lesson) { // skip if value is null because there weren't mult choice lessons in tutorial
+      logList(`Great! You've chosen to build a quiz for`, [`Tutorial: ${tutorial.title}`, `Lesson: ${lesson.title}`])
+      const shouldOverridePristineQuiz = api.lessons.isQuizPristine(tutorial, lesson) && await promptOverwritePristineQuiz(tutorial, lesson)
+      const shouldOverrideQuiz = !shouldOverridePristineQuiz && await promptOverwriteQuiz(tutorial, lesson)
+      if (shouldOverridePristineQuiz || shouldOverrideQuiz) {
+        return createQuiz(tutorial, lesson, { createLesson, createTutorial, createResource })
+      } else {
+        logEditQuizManually(tutorial, lesson)
+      }
+    }
+  } else {
+    log.info('No worries. Please run the ProtoWizard again to create your tutorial and lesson. Once you have the necessary files, you can run this script again to build your quiz.')
+  }
+} // end createQuizIntro
```

Comment:
Let's add some spacing here to make it more readable? I suggest the following:

```suggestion
async function createQuizIntro ({ createLesson, createTutorial, createResource }) {
  log.info("Let's create a multiple-choice quiz! This will only work if you've already created the tutorial and lesson files.")

  if (await promptFilesReady()) {
    const tutorial = await selectTutorial('quiz', { createTutorial, createResource, createLesson, createQuiz })
    const lesson = await selectLesson(tutorial)
 
    if (lesson) { // skip if value is null because there weren't multiple choice lessons in tutorial
      logList(`Great! You've chosen to build a quiz for`, [`Tutorial: ${tutorial.title}`, `Lesson: ${lesson.title}`])

      const shouldOverridePristineQuiz = api.lessons.isQuizPristine(tutorial, lesson) && await promptOverwritePristineQuiz(tutorial, lesson)
      const shouldOverrideQuiz = !shouldOverridePristineQuiz && await promptOverwriteQuiz(tutorial, lesson)

      if (shouldOverridePristineQuiz || shouldOverrideQuiz) {
        return createQuiz(tutorial, lesson, { createLesson, createTutorial, createResource })
      } else {
        logEditQuizManually(tutorial, lesson)
      }
    }
  } else {
    log.info('No worries. Please run the ProtoWizard again to create your tutorial and lesson. Once you have the necessary files, you can run this script again to build your quiz.')
  }
} // end createQuizIntro
```

Reply:
I would also do this with the rest of the file. Just adding some spaces after variable declaration blocks helps, as well as after logs.